### PR TITLE
Alert component

### PR DESCRIPTION
**Pull request by malaykhakhar**
Created alert component

### CODE REVIEW

File: components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.alert-warning {
+  background: rgba(245, 222, 11, 0.22);
+  border: 1px solid #605c00;
+}
+
+.alert-info {
+  background: #ddeaff;
+  border: 1px solid #334b71;
```
Comment (yaswanthmyneni) [inline components/Alert/alert.css]: give CSS variables for colors.

File: components/Alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+@import url("/style.css");
+
+.container-alert {
```
Comment (yaswanthmyneni) [inline components/Alert/alert.css:3]: change this name to alert-container for consistency of naming.

Comment (yaswanthmyneni) [inline components/Alert/alert.css]: I think this is not part of your component styling. you can remove it from this file and keep it in style.css. Then you can use this class for all other components too.

**Comment by yaswanthmyneni** [comment]
you can merge after doing changes.

File: components/Alert/alert.css (hunk 2 above)
Comment (malaykhakhar) [reply components/Alert/alert.css]: Done

File: components/Alert/alert.css (hunk 1 above)
Comment (malaykhakhar) [reply components/Alert/alert.css]: Done

File: components/Alert/alert.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        </p>
+        <div class="container-alert">
+          <div class="alert alert-success">This is a simple success alert!</div>
+          <div class="alert alert-error">This is a simple error alert!</div>
+          <div class="alert alert-warning">This is a simple warning alert!</div>
+          <div class="alert alert-info">This is a simple info alert!</div>
+        </div>
+        <iframe src="https://carbon.now.sh/embed/yVQuBgsUv2q9BRSATGfX" style="width: 680px; height: 448px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+        </iframe>
+</iframe>
```
Comment (sri-rishi) [inline components/Alert/alert.html]: I think </iframe> is here is extra

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Other wise looks good

**Pull request merged**